### ISSUE TITLE
Docker setup fails with 'permission denied'

### ISSUE THREAD

**Issue by Meldiron** [offen/offen#710]
(Following [Offen Docker docs](https://hub.docker.com/r/offen/offen))

Steps to reproduce:
1. Get a 10$ DO droplet with Ubuntu
2. Install docker:
```
    1  curl -fsSL https://get.docker.com -o get-docker.sh
    2  sh get-docker.sh
```
3. Prepare Offen:
```
    3  docker volume create offen_data
    4  docker volume create offen_certs
    5  mkdir -p ~/.config
    6  touch ~/.config/offen.env
```
4. Run Offen:
```
docker run -it --rm \
  -v offen_data:/var/opt/offen \
  -v offen_certs:/var/www/.cache \
  -v ~/.config/offen.env:/etc/offen/offen.env \
  offen/offen:latest setup \
  -email me@mysite.com \
  -name mysite \
  -populate
```

This results in:
```
FATA[0000] Error sourcing runtime configuration          error="config: error persisting settings: config: error writing env file: open /etc/offen/offen.env: permission denied"
```

I tried a different folder locations, but that didn't help.

**Comment by m90** [comment]
Seems the instructions you are using are out of date since #575 

Could you try making the config file owned by the correct user (i.e. the non-root user that is running Offen inside the container) something like this:

```
sudo chown 10000:10001 ~/.config/offen.env
```
Let me know if that works for you so we can get the docs up to date again. Thanks!

**Comment by Meldiron** [comment]
Heyyy 👋
I appreciate the quick response. Changing the owner of the file solved the issue and the installation has now finished properly.

**Comment by m90** [comment]
I updated the docs in [495b776](https://github.com/offen/offen/commit/495b776f14494c145a3ef04c1eb011c8c7150bf5) - will close this issue when it's included in a release.

**Comment by m90** [comment]
This is now included in `v1.1.0`. Thanks for the heads up.
